pkg/flowkit/config/network.go

```diff
@@ -86,6 +86,13 @@ func DefaultTestnetNetwork() Network {
 	}
 }
 
+func DefaultSandboxNetwork() Network {
+	return Network{
+		Name: "sandboxnet",
+		Host: "secure.sandboxnet.nodes.onflow.org:9000",
```

Comment:
why is this secure.sandboxnet.nodes.onflow.org:9000 and `access.sandboxnet.nodes.onflow.org:9000` instead?

Reply:
I didn't see access in the notion list.
I can change to access.